assets/styles/_syna.scss

```diff
@@ -63,3 +63,13 @@ article .content {
     }
   }
 }
+
+pre {
+  padding: .75rem;
+}
+
+.bg-dark {
+  pre {
+    background-color: white;
```

Comment:
Weirdly enough bootstrap with it's hundreds of variables doesn't provide any variable or styling for code blocks. And the only play the code blocks are not readable are in a dark background. Since we don't control the rendering of code blocks (since user enters it in markdown) the only place to customize it is in css.